pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -216,6 +216,19 @@ pyramid_oereb:
         stroke_opacity: 0.6
         stroke_color: '#e60000'
         stroke_width: 5
+    type_mapping:
```

Comment:
Am I right that this change means we have to adapt the configuration with our french translations (or do the mapping in fme beforehand)?

Reply:
Yes exactly. You need to map the type inside your data to the federal defined types. The system allows multi mapping.

An Example. In your data both "Grundstück" and "Liegenschaft" means "RealEstate" than you can configure it like this:
```
type_mapping:
  - mapping: RealEstate
    type: Liegenschaft
  - mapping: RealEstate
    type: Grundstück
```

This way everyone can leave its AV data like it is and configure correct mapping instead :grimacing:

Reply:
And you don't *must* configure this. If it is not configured it uses the value from data.